src/components/formCovid/index.tsx

```diff
@@ -90,79 +90,91 @@ const FormCovid = (): React.ReactElement => {
+        <div className="grid pb-1 rounded-md shadow-xl p-9 bg-whiteAccent">
+          <p className="text-xl font-medium mb-7 text-navyBlue">
+            Vul je covid gegevens aan van{' '}
+            <Text property={FOAF.name.iri.value} />
+          </p>
+          <p className="flex content-center justify-center gap-1 font-medium text-green-700 mb-7">
+            {solidStore.status}
+          </p>
+          <form
+            onSubmit={handleSubmit}
+            className="grid content-center justify-center gap-7 mb-7 "
+          >
+            <select
+              className="pl-2 pr-2 border border-textColor rounded-xl bg-whiteAccent text-navyBlue"
+              onChange={handleCertificaatChange}
+              required
+              name="certificaten"
+              id="certificaten"
+              value={certificaat}
+            >
+              <option value="vaccinatiecertificaat">
+                Vaccinatiecertificaat
+              </option>
+              <option value="testcertificaat"> Testcertificaat </option>
+              <option value="herstelcertificaat">Herstelcertificaat </option>
+            </select>
 
-        <select
-          onChange={handleGroupChange}
-          value={group}
-          required
-          name="groep"
-          id="groep"
-        >
-          <option value="wheelhouse">Wheelhouse</option>
-          <option value="konsolidate"> Konsolidate </option>
-          <option value="craftworkz">Craftworkz </option>
-        </select>
-        {certificaat === 'testcertificaat' ||
-        certificaat === 'herstelcertificaat' ? (
-          <input
-            onChange={handleDateChange}
-            min="2020-12-12"
-            required
-            type="date"
-            id="geldigheidsperiode"
-          />
-        ) : (
-          <input
-            required
-            type="file"
-            accept="application/pdf"
-            onChange={handleFilesChange}
-            id="covidfile"
-            name="covidfile"
-          />
-        )}
-        {file ? (
-          <CovidInformation
-            certificaat={certificaat}
-            date={date}
-            id={id}
-            dosis={dosis}
-            group={group}
-          />
-        ) : null}
-        <input
-          type="submit"
-          className="p-5 font-semibold text-white bg-indigo-700 rounded-sm w-44"
-          value="Save information"
-        />
-      </form>
-    </>
+            <select
+              className="pl-2 pr-2 border border-textColor rounded-xl bg-whiteAccent text-navyBlue"
+              onChange={handleGroupChange}
+              value={group}
+              required
+              name="groep"
+              id="groep"
+            >
+              <option value="wheelhouse">Wheelhouse</option>
+              <option value="konsolidate"> Konsolidate </option>
+              <option value="craftworkz">Craftworkz </option>
+            </select>
```

Comment:
Hier zou ik ook een apart React Componentje van maken. Is makkelijk als je dan later nog een Select ergens nodig hebt en je dat hetzelfde wilt stylen